dashboard/settings.py

```diff
@@ -45,7 +45,12 @@
 
 # Resource values from env
 RESOURCE_VALUES = ENV.get("RESOURCE_VALUES", {"files": {"types": ["canvas"], "icon": "fas fa-file fa-lg"}})
-RESOURCE_URLS = ENV.get("RESOURCE_URLS", {"canvas": {"prefix": "https://demo.instructure.com/files/", "postfix": "/download?download_frd=1"}})
+
+# Convience map to be able to get from types
+RESOURCE_VALUES_MAP = {type : value
+    for value in RESOURCE_VALUES
+    for type in RESOURCE_VALUES.get(value).get('types')
+}
```

Comment:
I'd like to see `type` renamed here. It works fine, but since `type` is a built into Python, it seems less confusing to use something like `resource_type`.

Reply:
Yeah, okay! I'll update this.